eng/pipelines/templates/jobs/ci.yml

```
@@ -336,14 +336,14 @@ jobs:
           arguments: >
             -Directory sdk/${{ parameters.ServiceDirectory }}
 
-      - ${{ if eq(parameters.EnableVerifyTypeSpecCodeGeneration, 'true') }}:
-        - task: PowerShell@2
-          displayName: Verify TypeSpec Code Generation
-          inputs:
-            pwsh: true
-            filePath: $(Build.SourcesDirectory)/eng/scripts/TypeSpec-Compare-CurrentToCodegeneration.ps1
-            arguments: >
-              -Directory sdk/${{ parameters.ServiceDirectory }}
+      - task: PowerShell@2
+        displayName: Verify TypeSpec Code Generation
+        continueOnError: ${{ if eq(parameters.EnableVerifyTypeSpecCodeGeneration, 'false') }}
```

We don't need `if` inside `${{ }}`
```suggestion
        continueOnError: ${{ eq(parameters.EnableVerifyTypeSpecCodeGeneration, 'false') }}
```